dev-docs/modules/userId.md

```diff
@@ -165,6 +166,38 @@ pbjs.setConfig({
 });
 {% endhighlight %}
 
+### ID+
+
+ID+, a zeotap powered solution, is an open invitation to the entire ecosystem to collaboratively solve the industry challenge of identity resolution in a third-party cookieless environment. 
+
+This sub-module enables the user’s ID+ to be available in the bid request.
+
+More information on ID+ can be found here: [https://idplus.io/](https://idplus.io/)
+
+Add it to your Prebid.js package with:
+
+{: .alert.alert-info :}
+gulp build --modules=zeotapIdPlusIdSystem
+
+#### ID+ Registration
+
+You can set up your ID+ account by contacting our support team at [support.idplus@zeotap.com](mailto:support.idplus@zeotap.com) or via [https://idplus.io/contact-us](https://idplus.io/contact-us.html) and we will get back to you.
+
+ID+ is covered under zeotap privacy policy: [Zeotap Privacy Policy](https://zeotap.com/website-privacy-policy).
```

Comment:
Same question about the capitalization.

Reply:
Same as the above, used capitalization here only for 'Zeotap Private Policy'